test/plugins/validation/2and3/security-definitions-ibm.js

```diff
@@ -115,13 +116,40 @@ describe('validation plugin - semantic - security-definitions-ibm', function() {
         }
       };
 
-      const res = validate({ jsSpec: spec }, config);
+      const res = validate({ resolvedSpec: spec }, config);
       expect(res.errors.length).toEqual(0);
       expect(res.warnings.length).toEqual(0);
     });
   });
 
   describe('OpenAPI 3', function() {
+    it('should warn about an unused security definition', async function() {
```

Comment:
This name is also used by the test below it - we should not have duplicated test messages. Will you change this to `should follow references to security schemes` ?